### DISCUSSION TITLE
exclude MS-SQL system table 'sysdiagrams'

### DISCUSSION THREAD

**Discussion by paulboudewijn** [directus/directus#9826] [Feature Requests]
In MS-SQL the (system) table `sysdiagrams` is automatically generated when you enable the database diagrams functionality.
This table gets listed in Directus:

![image](https://user-images.githubusercontent.com/17253180/141841262-fd6f77a2-f8e0-4a1d-98a3-129ed2800b4f.png)

Is it okay to add this to the default excluded tables list? I've got a PR ready.

**Comment by rijkvanzanten** [comment]
> Is it okay to add this to the default excluded tables list? I've got a PR ready.

Yes, totally!
